metadata.rb

```diff
@@ -16,4 +16,4 @@
 supports 'suse'
 supports 'ubuntu'
 
-depends 'ark', '~> 5.0.0'
+depends 'ark', '~> 6.0.0'
```

Comment:
Shouldn't we have a looser dep on this?
```suggestion
depends 'ark', '>= 6.0'
```

Reply:
I go with only pinning to major versions generally so I'm ok with that but I'd rather use the fuzzy match or pin it` >=6 <7`. 

I know it's been a hot topic recently but looking at it like a gem dependency I can't see how leaving it open on the upper band is any better.

Reply:
If you want to lock just the major then you could do `~> 6.0`. If you do `6.0.0` you'll force users to use `6.0` in the long run which I don't think is ideal with sous-chefs cookbooks.